src/lib/pbo/quadratization.jl

```diff
@@ -12,7 +12,7 @@ end
 @doc raw"""
     quadratize!(aux::Function, f::PBF{S, T}, ::Quadratization{Q}) where {S,T,Q}
 
-Quadratizes a given PBF in-place, i.e. applies a mapping ``Q : \mathscr{F}^{k} \to \mathscr{F}^{2}``.
+Quadratizes a given PBF in-place, i.e. applies a mapping ``Q : \mathscr{F}^{k} \to \mathscr{F}^{2}``, where Q is the quadratization method.
```

Comment:
We are using Q for the QUBO definition. Maybe use \mathcal{Q} for quadratization